apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -120,6 +120,7 @@ cluster(["join", SNode]) ->
             emqx_ctl:print("Failed to join the cluster: ~0p~n", [Error])
     end;
 cluster(["leave"]) ->
+    emqx_cluster_rpc:on_leave(),
     case ekka:leave() of
         ok ->
```

Comment:
What happen if ekka:leave failed but we already delete cluster_commit table ?

Reply:
we should also add delete cluster_commit when `force-leave Node` cli on line 132.
```
cluster(["force-leave", SNode]) ->
```

Reply:
> What happen if ekka:leave failed but we already delete cluster_commit table ?

I have saw the ekka:leave code, it only fails if the node don'ts in the cluster, so I think here is safe.